cli/src/alluxio.org/cli/cmd/quorum/elect.go

```diff
@@ -12,27 +12,27 @@
 package quorum
 
 import (
+	"github.com/palantir/stacktrace"
 	"github.com/spf13/cobra"
 
 	"alluxio.org/cli/env"
 )
 
 var Elect = &ElectCommand{
-	BaseJavaCommand: &env.BaseJavaCommand{
-		CommandName:   "elect",
-		JavaClassName: "alluxio.cli.fsadmin.FileSystemAdminShell",
-		Parameters:    []string{"journal", "quorum"},
+	QuorumCommand: &QuorumCommand{
+		BaseJavaCommand: &env.BaseJavaCommand{
+			CommandName:   "elect",
+			JavaClassName: "alluxio.cli.fsadmin.FileSystemAdminShell",
+			Parameters:    []string{"journal", "quorum", "elect"},
+		},
+		allowedDomains: []string{domainMaster},
 	},
 }
 
 type ElectCommand struct {
-	*env.BaseJavaCommand
-
-	ServerAddress string
-}
+	*QuorumCommand
 
-func (c *ElectCommand) Base() *env.BaseJavaCommand {
-	return c.BaseJavaCommand
+	serverAddress string
```

Comment:
can use the `hostname_port` validator for this as well
```suggestion
	serverAddress string `validate:"hostname_port"`
```